### ISSUE TITLE
Pretrained model or training from scratch for a large Xenium dataset?

### ISSUE THREAD

**Issue by pfren1998** [MICS-Lab/novae#68]
Hi Novae team,

Thank you for developing this great tool.

I have 130 Xenium 5K samples with about 50 million cells in total. For this dataset, would you recommend using the pretrained model, fine-tuning it on some representative slides, or training a new model from scratch?

Since all samples are from the Xenium 5K platform, I am wondering whether training from scratch would provide a clear advantage. If fine-tuning is recommended, how many high-quality slides would usually be sufficient?

Thank you for your advice!

**Comment by quentinblampey** [comment]
Hello @pfren1998,

I definitely recommend fine-tuning the pretrained model, this should be more stable and also more performant.
Even better: if you can, using [this Novae+scConcept model](https://prism-oncology.github.io/novae/tutorials/novae_scConcept/) is usually much better.

> If fine-tuning is recommended, how many high-quality slides would usually be sufficient?

I recommend using only top-quality slides. You don't necessarily need to use a lot of slides, as long as they are relatively representative of your dataset. This depends a lot on your specific cohort, of course, but given your dataset size, I would say something between 5 and 20 slides should be great!

Don't hesitate to show me your loss plot and/or the final entropy after the fine-tuning if you want some advice or a double-check that the fine-tuning went well :)

**Comment by pfren1998** [comment]
Thanks for your suggestion! I will try that approach. And I want to make sure that the final domains are not affected by the batch effects and are comparable across samples. Is it right?

**Comment by quentinblampey** [comment]
> I want to make sure that the final domains are not affected by the batch effects and are comparable across samples. Is it right?

Yes, exactly, it's one of the main reasons why we wanted to develop Novae 😊 Even if you save the model and re-use it later on new slides, you'll find the same domains again, see [this docs section](https://prism-oncology.github.io/novae/tutorials/novae_scConcept/#continuous-data-generation)

You may be interested in [this section](https://prism-oncology.github.io/novae/faq/#how-to-avoid-overcorrecting) of the docs.

**Comment by pfren1998** [comment]
Hi, thanks for this suggestion. Our spatial transcriptomics data are being generated continuously, and the cohort includes multiple organs. Some future slides may even come from completely new organs.

If we want to analyze each slide as soon as it becomes available, what is the recommended workflow to keep spatial domains comparable across batches?

Would you recommend fixing the prototypes from an initial set of slides and applying the saved model to new slides? If a new organ contains genuinely novel domains, how can we allow new domains to emerge without losing comparability with previously analyzed slides?

Thanks!

**Comment by quentinblampey** [comment]
Hello @pfren1998,

You can definitely keep the same model for all the new samples (as in the tutorial mentioned above), but it will not perform well at detecting domains/niches specific to new organs it didn't see during the first fine-tuning.

If you start to have many samples for a new organ, I would consider fine-tuning a new Novae model again. I know this is not very convenient, so ideally you shouldn't do it very often...

As an example use case: we have similar issues on our side, and we started early with an initial model, then made other versions when new organs were introduced and in sufficient quantity. To avoid having 10 different models, we waited for a sufficient number of samples from the new organ before fine-tuning again, and now that we have 100+ samples we have decided to keep the model we had and not fine-tune anymore.

**Comment by pfren1998** [comment]
Thank you for your timely reply!
